withdraw-by-check/withdraw-by-check-json-contract.md

```diff
@@ -108,10 +108,30 @@ If the request is not successful due to insufficient funds, the poweron should r
 }
 ```
 
+If the request is not successful due to a missing address, the poweron should respond with:
+
+```json
+{
+  "errorCode": 502,
+  "loggingErrorMessage": "TRANPERFORM Error:+TRANERROR"
+}
```

Comment:
How is this error code situation (503) different than the error code situation described on line 55?

Reply:
Good catch. Looks like a duplicate. I'll adjust.